docs/rest.md

```diff
@@ -125,152 +142,149 @@ string. For example, `"1234567891234567"` represents
       "name": "ubuntu-core",
       "origin": "ubuntu",
-      "resource": "/1.0/packages/ubuntu-core.ubuntu",
+      "resource": "/2.0/snaps/ubuntu-core.ubuntu",
       "status": "active",
-      "type": "core",
+      "type": "os",
       "update_available": "247",
       "version": "241"
     }
- }
+ },
+ "paging": {
+    "count": 3,
+    "page": 0,
+    "pages": 1
+  },
+  "sources": [
+    "local",
+    "store"
+  ]
 }
 ```
 
 #### Fields
-* `packages`
+* `snaps`
     * `status`: can be either `not installed`, `installed`, `active` (i.e. is
       current), `removed` (but data present); there is no `purged` state, as a
-      purged package is undistinguishable from a non-installed package.
-    * `name`: the package name.
+      purged snap is undistinguishable from a non-installed snap.
+    * `name`: the snap name.
     * `version`: a string representing the version.
-    * `icon`: a url to the package icon, possibly relative to this server.
-    * `type`: the type of snappy package; one of `app`, `framework`, `kernel`,
+    * `icon`: a url to the snap icon, possibly relative to this server.
+    * `type`: the type of snap; one of `app`, `framework`, `kernel`,
       `gadget`, or `os`.
-    * `description`: package description
-    * `installed_size`: for installed packages, how much space the package
-      itself (not its data) uses, formatted as a decimal string.
-    * `download_size`: for not-installed packages, how big the download will
+    * `description`: snap description
+    * `installed_size`: for installed snaps, how much space the snap
+      itself (not its data) uses.
+    * `download_size`: for not-installed snaps, how big the download will
       be, formatted as a decimal string.
-    * `operation`: if the state signals that an operation is underway
-      (e.g. installing), the operation field describes that operation
-    * `rollback_available`: if present and not empty, it means the package can
+    * `rollback_available`: if present and not empty, it means the snap can
       be rolled back to the version specified as a value to this entry.
-    * `update_available`: if present and not empty, it means the package can be
+    * `update_available`: if present and not empty, it means the snap can be
       updated to the version specified as a value to this entry.
+* `paging`
+    * `count`: the number of snaps on this page
+    * `page`: the page number, starting from `0`
+    * `pages`: the (approximate) number of pages
+* `sources`
+    a list of the sources that were queried (see the `sources` parameter, below)
+
+### Parameters [fixme: is that the right word for these?]
+
+#### `sources`
+
+Defaults to `local,store`; can be set to either `local` (to only list
+local snaps) or `store` (to only list snaps from the store), or a
+comma-separated combination.
+
+#### `page`
+
+Defaults to `0`, request the given page when the server is paginating the
+result.
```

Comment:
I find it more readable when the default value is shown at the end.
Like:
Request the given page when the server is paginating the result. Defaults to `0`.